edx_proctoring/signals.py

```diff
@@ -105,6 +106,16 @@ def on_attempt_changed(sender, instance, signal, **kwargs):  # pylint: disable=u
             # to archive it
             original = sender.objects.get(id=instance.id)
 
+            # if the exam was just completed for the first time, we want to mark it as complete
+            # in the completion service
```

Comment:
A) you use completed on both sides of this sentence - is `completed` also a term of art in the proctoring djangoapp? Maybe `attempted` would work better?
B) Might be worth mentioning _why_ we have custom completion logic - that we are worried about timing out and such?

Reply:
A) I'll look into rephrasing. I think I used `completed` because of the field we are looking at being updated.
B) Sure thing